tools/scoring/winners.csv

```diff
@@ -14,7 +14,7 @@ LRA,CVC4,CVC4,CVC4 Z3,CVC4 Z3
 NIA,CVC4 Z3,ProB Z3,CVC4 Z3,CVC4 Z3
 NRA,CVC4,Vampire,Redlog,Vampire Z3 / Vampire
 QF_ABV,Boolector,Boolector,Boolector,Boolector 
-QF_ABVFP,,,---,CVC4
+QF_ABVFP,,,CVC4

```

Comment:
The reason why there was '---' in there is that in 2017 we had no entrants to that division. Shouldn't make a difference, though.

Reply:
Shouldn't make a difference for this check.